src/site/content/en/learn/css/selectors/index.md

```diff
@@ -0,0 +1,486 @@
+Use `::marker` to style each bullet point (or number) in the list
+
+```css
+/* Your list will now either have red dots, or red numbers */
+li::marker {
+	color: red;
+}
+```
+
+You can also use `::selection` to style the content that has been highlighted by a user.
+
+```css
+::selection {
+	background: black;
+	color: white;
+}
+```
+
+You can find out more about pseudo-elements in a later lesson.
+
+## Complex selectors
+
+You have already seen a vast array of selectors,
+but sometimes, you will need more *fine-grained control* with your CSS.
+This is where complex selectors step in to help.
+
+It's worth remembering at this point that although the following selectors give us more power,
+we can only  **cascade downwards**, selecting child elements.
+We are not able to target upwards and select a parent element.
+We cover what the cascade is and how it works in a later lesson.
+
+### Combinators
+
+A combinator is what sits between two selectors.
+For example, if the selector was `p > strong`, the combinator is the `>` character.
+The selectors which use these combinators help you select items based on their position in the document.
+
+#### Descendant combinator
+
+To understand descendant combinators,
+you need to first understand parent and child elements.
+
+```html
+<p>
+  A paragraph  of text with some <strong>bold text for emphasis</strong>.
+</p>
+```
+
+The parent element is the `<p>` which contains text.
+Inside that `<p>` element is a `<strong>` element, making its content bold.
+Because it is inside the `<p>`, it is a child element.
+
+A descendant combinator allows us to target a child element.
+This uses a space (` `) to instruct the browser to look for child elements:
+
+```css
+p strong {
+	color: blue
+}
+```
+
+This snippet selects all `<strong>` elements
+that are child elements of `<p>` elements only and makes them blue.
+
+A combinator selector like this is **recursive**.
+This means that in the above snippet, all strong elements,
+regardless of their depth, will have blue text.
+
+<figure class="w-figure">
+  {% Codepen {
+    user: 'web-dot-dev',
+    id: 'BapBbGN'
+  } %}
+  <figcaption class="w-figcaption">Because the descendant combinator is recursive,
+the padding added to each child element applies, resulting in a staggered effect.</figcaption>
+</figure>
+
+This effect is better visualised in the above example,
+using the combinator selector, `.top div`.
+That CSS rule adds left padding to those `<div>` elements.
+Because the combinator is recursive,
+all `<div>` elements that are in `.top` will have that same padding applied to them.
+
+Take a look at the HTML panel in this demo
+to see how the `.top` element has several `<div>` child elements which themselves,
+have `<div>` child elements.
+
+#### Next sibling combinator
+
+You can look for an element that immediately follows another element
+by using a `+` character in your selector.
+
+{% Codepen {
+    user: 'web-dot-dev',
```

Comment:
I think this demo can be more noticeable. Maybe make the borders of the children a different color, like red to really highlight the differences (margin top AND border: red)

Reply:
I've added a little pseudo-element as a highlighter. Works well!